src/GenericFactory.sol

```diff
@@ -195,22 +191,16 @@ contract GenericFactory is IGenericFactory, Owned {
 
     event Deployed(bytes32 codeId, address _address);
 
-    /// @notice Deploys a given bytecode with provided token0 & token1 args.
-    function deploy(bytes32 aCodeKey, address aToken0, address aToken1) external payable returns (address rContract) {
-        require(_deployInProgress, "FACTORY: ONLY_CHILDREN_CAN_CALL");
-
+    function deploySharedContract(bytes32 aCodeKey, address aToken0, address aToken1) external onlyOwner returns (address rContract) {
         bytes memory lInitCode = getBytecode(aCodeKey, aToken0, aToken1);
 
         // SAFETY:
         // Does not write to memory
-        assembly ("memory-safe") {
-            // sanity checked against OZ implementation:
-            // https://github.com/OpenZeppelin/openzeppelin-contracts/blob/3ac4add548178708f5401c26280b952beb244c1e/contracts/utils/Create2.sol#L40
+        assembly("memory-safe") {
             rContract := create2(callvalue(), add(lInitCode, 0x20), mload(lInitCode), 0)
 
-            if iszero(extcodesize(rContract)) { revert(0, 0) }
+            if iszero(extcodesize(rContract)) { revert(0,0) }
         }
-
         emit Deployed(aCodeKey, rContract);
     }
```

Comment:
What happened here with the formatting?

- The space between assembly and memory-safe
- The newline before emitting the event
- The spacing on revert

Reply:
my bad, I was doing this by hand and actually had another function. 

Anyway, just fixed it.